docs/en/status-communities/invite-people-to-a-channel.md

```diff
@@ -0,0 +1,55 @@
+---
+id: 286
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Invite people to a channel
+
+Whether you're a channel owner or a member, you can tell people about the channel and invite them to join.
+
+!!! note
+    You can't join a channel if you are not a member of a community.
```

Comment:
To join the channel, the person you're inviting needs to be a member of the community.